### DISCUSSION TITLE
Switching from "Rolling Release" to "Stable"

### DISCUSSION THREAD

**Discussion by KoneSV** [AzuraCast/AzuraCast#4102] [Q&A]
How can I change my version of  "Rolling Release" to "Stable"
I use the Ansible version

**Comment by KoneSV** [comment]
Ansible version?El 30 abr. 2021 2:05 p. m., zloysystem ***@***.***> escribió:
https://docs.azuracast.com/en/getting-started/updates/release-channels

**Comment by KoneSV** [comment]
Why close my post, without giving me a solution?

**Reply by Vaalyn** [reply]
I have just moved your question from the Issues list to the Discussions since we want to mostly keep the issue list for bugs, errors and such things so that we can better keep track of those things.

Please make sure that the rolling-release version you are currently on is older than the newest stable release version. You should only change from rolling-release to stable if the newest stable version is actually newer than the one you are currently using.

I'm not very experienced with the Ansible installation method so I'm not 100% sure on these commands but I think you'll need to run the following:

```
cd /var/azuracast/www
git checkout -q -f stable
```

And then run the update steps:

```
chmod a+x update.sh
./update.sh
```

You might have to run these with `sudo` if you get permission errors.

**Reply by KoneSV** [reply]
Lo hice y me quedo: 502 Bad Gateway

**Comment by KoneSV** [comment]
That documentation doesn't help me for Ansible
